packages/react/src/patterns/sub-patterns/CardLink/CardLink.js

```diff
@@ -47,6 +48,17 @@ const CardLink = ({
         {renderFooter(icon)}
       </div>
     </ClickableTile>
+  ) : (
+    <Tile
+      data-autoid={`${stablePrefix}--card-link`}
```

Comment:
I see the stable selectors, file names, imports, and CSS classnames are still `cardlink`. Should we rename the component to just `card`?

Reply:
yeah let's do that